doc/src/modules/assumptions/index.rst

```diff
@@ -32,7 +32,7 @@ assumptions about objects in expr. Valid values include:
     * Q.integer(x) & Q.positive(x)
     * etc.
 
-Q is a class in sympy.assumptions holding known predicates.
+Q is an instance of QClass in sympy.assumptions.ask holding known predicates.
```

Comment:
QClass is just a class for Q. Q used to be a class itself but it needed to be made into an object so that the attributes could be made into properties. 

Q itself is not a very descriptive name. It's designed to be easy to type. We should probably take this opportunity to give the class a descriptive name for what Q actually is. Something like `AssumptionKeys`.